recipes/pysbol/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pysbol" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/pySBOL/archive/{{ version }}.tar.gz
+  sha256: 65d79b956e1249370307bc2a1e8638f776bd10b0559a6ab507477a65bcedc74c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+
+requirements:
+  host:
+    - python =3.7
```

Comment:
This is quite restrictive especially for a package that provides a library (bindings in this case). As an application developer, I would not want to be restricted to Python 3.7.

In generaly, applications can have quite strict dependencies, but libraries should be quite open.